src/nblint.py

```diff
@@ -22,13 +24,14 @@ def main():
         check(notebook, checks)
 
 
-def find_notebooks(dir):
+def find_notebooks(dir, ignore_paths):
     notebooks = set()
-    for dirname, _, filenames in os.walk(dir):
+    for rootname, dirnames, filenames in os.walk(dir, topdown=True):
```

Comment:
I'd suggest sticking to the conventional `(dirpath, dirnames, filenames)`, matching the Python docs.

Reply:
Or `dirpath, _, filenames` if we don't need that second element of the tuple